language/move-analyzer/src/symbols.rs

```diff
@@ -2190,6 +2198,228 @@ pub fn on_hover_request(context: &Context, request: &Request, symbols: &Symbols)
     );
 }
 
+/**
+ * Returns a tuple where the first value is the index of the text info of the function name and the second value is the text info of the function
```

Comment:
Please add explanation of "text info" for the function describes (function name and position/range of the name?)